### PR TITLE
Update ThreeDSecureIO3dsServer settings

### PR DESCRIPTION
**Pull request by ashkarpetin**
Add:
acquirerMerchantIdVisa 
acquirerMerchantIdMastercard

Remove:
merchantId

### CODE REVIEW

File: openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml [hunk 1]
```diff
@@ -5,16 +5,21 @@ allOf:
...
       - merchantAcquirerBinVisa
       - merchantAcquirerBinMastercard
       - merchantCountry
       - merchantUrl
     properties:
-      merchantId:
+      acquirerMerchantIdVisa:
         type: string
-        description: Merchant Id.
+        description: Visa acquirer MID.
```
Comment (ashkarpetin) [inline openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml:18]:
```suggestion
        description: Visa acquirer merchant ID.
```

File: openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml [hunk 2]
```diff
@@ -5,16 +5,21 @@ allOf:
...
+        maxLength: 35
+      acquirerMerchantIdMastercard:
+        type: string
+        description: Mastercard acquirer MID.
```
Comment (ashkarpetin) [inline openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml:22]:
```suggestion
        description: Mastercard acquirer merchant ID.
```

**Comment by ashkarpetin** [comment]
php please review

**Review by justnero — DISMISSED** [review]

**Comment by ashkarpetin** [comment]
php please review

File: openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml [hunk 3]
```diff
@@ -5,16 +5,21 @@ allOf:
...
       - merchantAcquirerBinVisa
       - merchantAcquirerBinMastercard
       - merchantCountry
       - merchantUrl
     properties:
-      merchantId:
+      acquirerMerchantIdVisa:
         type: string
-        description: Merchant Id.
+        description: Visa acquirer merchant ID.
```
Comment (colinbird) [inline openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml:18]:
```suggestion
        description: Visa acquirer merchant ID (MID).
```

File: openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml [hunk 4]
```diff
@@ -5,16 +5,21 @@ allOf:
...
+        maxLength: 35
+      acquirerMerchantIdMastercard:
+        type: string
+        description: Mastercard acquirer merchant ID.
```
Comment (colinbird) [inline openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml:22]:
```suggestion
        description: Mastercard acquirer merchant ID (MID).
```

**Review by colinbird — APPROVED** [review]

**Comment by ashkarpetin** [comment]
php please re-review

**Comment by ashkarpetin** [comment]
php please re-review

**Review by XaosSintez — APPROVED** [review]

**Pull request merged**